### ISSUE TITLE
Allow searching with multiple keywords

### ISSUE THREAD

**Issue by saeedesmaili** [Acidham/chromium-hist-bookmarks#32]
Thanks for this awesome workflow.
Assuming I have an item in my history which titled "The sample projectX design doc", if I search for "projectX doc" it doesn't appear in the search results, because "projectX" and "doc" are not immediately after each other in the title of the history item. I think it would make sense to display any item in the history that has both "projectX" and "doc" in the title.

**Comment by Acidham** [comment]
Yes, that is correct and expected. Per documentation, you need to use `&` operator `projectX&doc`
